cpp/inc/bond/ext/grpc/win_thread_pool.h

```diff
@@ -25,7 +25,7 @@ namespace bond { namespace ext { namespace grpc
 {
     namespace detail
     {
-        [[noreturn]] void Win32Exception(const char* message)
+        inline [[noreturn]] void Win32Exception(const char* message)
```

Comment:
Minor: the rest of the Bond code uses `[[noreturn]] inline` in that order:

```suggestion
        [[noreturn]] inline void Win32Exception(const char* message)
```

Reply:
Changed order and force pushed an update.